From 5eae213031a3ef88fe8d9d0a387f44264ea635cc Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 5 Oct 2013 15:53:16 +0200
Subject: [PATCH 2/3] network/ifconfig: include <linux/if_slip.h> instead of
 <net/if_slip.h>

The musl C library doesn't provide the <net/if_slip.h> since userspace
applications can just as well use the <linux/if_slip.h> kernel header.

This commit fixes the build of the ifconfig applet with the musl C
library.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 networking/ifconfig.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/networking/ifconfig.c b/networking/ifconfig.c
index 782374b..c90ed6b 100644
--- a/networking/ifconfig.c
+++ b/networking/ifconfig.c
@@ -56,7 +56,7 @@
 #endif
 
 #if ENABLE_FEATURE_IFCONFIG_SLIP
-# include <net/if_slip.h>
+# include <linux/if_slip.h>
 #endif
 
 /* I don't know if this is needed for busybox or not.  Anyone? */
-- 
1.8.1.2

